test/tests/smoke_test.rs

```diff
@@ -481,13 +481,10 @@ fn test_fips<T: HwModel>(hw: &mut T, fmc_version: u32, app_version: u32) {
     let resp = hw.mailbox_execute(u32::from(CommandId::SHUTDOWN), payload.as_bytes());
     assert!(resp.is_ok());
 
-    // Check we are rejecting additional commands with the shutdown error code.
-    let expected_err = Err(ModelError::MailboxCmdFailed(0x000E0008));
-    let payload = MailboxReqHeader {
-        chksum: caliptra_common::checksum::calc_checksum(u32::from(CommandId::VERSION), &[]),
-    };
-    let resp = hw.mailbox_execute(u32::from(CommandId::VERSION), payload.as_bytes());
-    assert_eq!(resp, expected_err);
+    // Check we are rejecting additional commands.
+    let _ = hw
```

Comment:
might be good to check that the error code is the one we expect (and we don't happen to be failing for some other reason)

Reply:
I also think it might be worthwhile to test that several commands fail (like calling this in a loop 3 times).

I could see a regression which causes the next command to fail, but then Caliptra goes back into a normal state.

Reply:
Done. I am sending different commands now and checking for shutdown error. Note this sequence exercised by rom and rt to make sure we are aligned in behavior.